libraries/AP_DDS/README.md

```diff
@@ -231,7 +231,32 @@ Next, follow the associated section for your chosen transport, and finally you c
   ```
   In order to consume the transforms, it's highly recommended to [create and run a transform broadcaster in ROS 2](https://docs.ros.org/en/humble/Concepts/About-Tf2.html#tutorials).
 
+## Using ROS 2 services (with Integration Services)
 
+### Prerequisites
+- Install and setup [Micro-XRCE Agent](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html#installing-the-agent-standalone)
+- Install and setup [Integration Services](https://micro-xrce-dds.docs.eprosima.com/en/latest/installation.html#installing-the-client-standalone) (it would be good to have a seperate workspace for this)
```

Comment:
Is this the right link? It's not the integration service.

Reply:
fixed the link..thanks for pointing out